testsuite/features/core/proxy_branch_network.feature

```diff
@@ -21,6 +21,14 @@ Feature: Setup SUSE Manager for Retail branch network
     And I manually install the "bind" formula on the server
     And I synchronize all Salt dynamic modules on "proxy"
 
+@proxy
+@private_net
+  # WORKAROUND
+  Scenario: Remove dhcp packages on the proxy
+    When I remove package "dhcp" from this "proxy"
+    And I remove package "dhcp-client" from this "proxy"
```

Comment:
maybe `When I remove package "dhcp dhcp-client" from this "proxy"` would have worked. If i remember correctly, you can pass a packages list.

Reply:
Done, thanks :-)